### PR TITLE
Feat: create 2 screens: deposit with card and phone»

### PR DESCRIPTION
**Pull request by liubaterehova**

### CODE REVIEW

File: src/components/ConfirmDeposit/ConfirmDeposit.jsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  subTitle,
+  navigation,
+  leftIcon,
+}) {
+  const [isDisabled, setDisabled] = useState(true);
+  const inputRef = useRef();
+  const currency = '₣';
+
+  useEffect(() => {
+    if (inputRef.current) inputRef.current.focus();
```
Comment (apsevkin) [inline src/components/ConfirmDeposit/ConfirmDeposit.jsx:39]: use autoFocus props for TextInput instead useEffect

File: src/components/ConfirmDeposit/ConfirmDeposit.jsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    if (navigation.canGoBack()) {
+      navigation.goBack();
+    }
+  }, [navigation]);
+
+  const handleChangeText = useCallback(
+    (text) => {
+      const processedMoney = processMoney(text);
+      onChangeAmountMoney(processMoney(processedMoney));
+      if (processedMoney) {
```
Comment (apsevkin) [inline src/components/ConfirmDeposit/ConfirmDeposit.jsx:52]: setDisabled(!processMoney);

File: src/screens/ConfirmMobileDeposit/ConfirmMobileDeposit.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import Text from 'components/Text';
+import ConfirmDeposit from 'components/ConfirmDeposit';
+
+import phoneOperator from 'assets/icons/MtnPhoneOperator.svg';
+
+import styles from './ConfirmMobileDeposit.styles';
+
+function ConfirmMobileDeposit({ navigation }) {
+  const [value, onChangeText] = useState('');
+  // const { phoneOperator, phoneNumber } = route.params;
```
Comment (apsevkin) [inline src/screens/ConfirmMobileDeposit/ConfirmMobileDeposit.jsx:14]: .

File: src/screens/ConfirmCardDeposit/ConfirmCardDeposit.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+import Text from 'components/Text';
+import ConfirmDeposit from 'components/ConfirmDeposit';
+
+import cardPayment from 'assets/icons/mastercard.svg';
+
+import styles from './ConfirmCardDeposit.styles';
+
+function ConfirmCardDeposit({ navigation }) {
+  const [value, onChangeText] = useState('');
+  // const { cardPayment, cardName, cardExpirationDate, cardNumber } = route.params;
```
Comment (apsevkin) [inline src/screens/ConfirmCardDeposit/ConfirmCardDeposit.jsx:14]: .

**Review by apsevkin — CHANGES_REQUESTED** [review]

File: src/components/ConfirmDeposit/ConfirmDeposit.jsx (hunk 1 above)
Comment (liubaterehova) [reply src/components/ConfirmDeposit/ConfirmDeposit.jsx:39]: I used it. But in ios platform it adds to keyboard padding and we can avoid it only if I make like this.

**Review by apsevkin — APPROVED** [review]

**Pull request merged**